packages/webapp-manager-deb/webapp-manager-deb.pacscript

```diff
@@ -2,7 +2,7 @@ name="webapp-manager-deb"
 gives="webapp-manager"
 repology=("project: webapp-manager")
 version="1.1.9"
-url="https://packages.linuxmint.com/pool/main/w/webapp-manager/webapp-manager_${version}_all.deb"
+url="http://packages.linuxmint.com/pool/main/w/webapp-manager/webapp-manager_${version}_all.deb"
```

Comment:
Why?

Reply:
Because try going to `https://packages.linuxmint.com/pool/main/w/webapp-manager` then `http://packages.linuxmint.com/pool/main/w/webapp-manager`. Going with http connection doesn't work as reported by Zahrun https://github.com/pacstall/pacstall-programs/pull/826#issuecomment-1077704158.

Reply:
Oh Okay.